src/lib/Microsoft.Health.Extensions.Fhir.R4/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Health.Extensions.Host.Auth;
+using Microsoft.Health.Logging.Telemetry;
+using FhirClient = Microsoft.Health.Fhir.Client.FhirClient;
+using IFhirClient = Microsoft.Health.Fhir.Client.IFhirClient;
+
+namespace Microsoft.Health.Extensions.Fhir
+{
+    public static class ServiceCollectionExtensions
```

Comment:
Good point, these methods are being used for doing DI on the FhirClient. I think if we want to go ahead and refactor the FhirService.cs class to extension methods then we could refactor these methods into that class. Not sure if it's not necessary otherwise

Happy to discuss further